packages/dnb-eufemia/src/extensions/forms/Field/Date/Date.tsx

```diff
@@ -73,8 +87,19 @@ function DateComponent(props: Props) {
     handleFocus,
     handleBlur,
     handleChange,
+    range,
   } = useFieldProps(preparedProps)
 
+  const rangeValue = useMemo(() => {
+    if (!range) {
+      return
+    }
+
+    const [startDate, endDate] = value.split(/\s/)
```

Comment:
```suggestion
    const [startDate, endDate] = value.split(' ')
```
We don't need a regex here. Then I think we should not use it.

Reply:
Could replace this with the pipe-character if we decide to go with that approach